src/lib/components/workflow/workflows-summary-configurable-table/workflows-summary-configurable-table.svelte

```diff
@@ -367,81 +394,64 @@
 
 <style lang="postcss">
   .workflow-summary-tables-wrapper {
-    @apply relative flex flex-row w-full rounded-lg border-primary border-2 overflow-scroll bg-white;
+    @apply relative flex flex-row w-full rounded-xl border-primary border-2 bg-white overflow-auto;
```

Comment:
Looks like there are a view overflow properties that are leading to this behavior (still looking for a solution) ⬇️ 

<img width="655" alt="Screenshot 2023-04-12 at 1 16 55 PM" src="https://user-images.githubusercontent.com/15069288/231561891-d68ac11d-79ce-4c0e-a49d-e54b56f6c6eb.png">

One is the `overflow-auto` here (see others below).